tests/test_inlinequery.py

```diff
@@ -68,6 +68,15 @@ def test(*args, **kwargs):
         monkeypatch.setattr(inline_query.bot, 'answer_inline_query', test)
         assert inline_query.answer()
 
+    def test_answer_auto_pagination(self, monkeypatch, inline_query):
+        def test(*args, **kwargs):
+            inline_query_id = args[0] == inline_query.id
+            offset = kwargs.get('current_offset') == inline_query.offset
+            return offset and inline_query_id
+
+        monkeypatch.setattr(inline_query.bot, 'answer_inline_query', test)
```

Comment:
```suggestion
    def test_answer_auto_pagination(self, monkeypatch, inline_query):
        def make_assertions(*args, **kwargs):
            inline_query_id = args[0] == inline_query.id
            offset = kwargs.get('current_offset') == inline_query.offset
            return offset and inline_query_id

        monkeypatch.setattr(inline_query.bot, 'answer_inline_query', make_assertions)
```

(in lack of a better name...)
In any case, don't name it `test`, that might be confusing (also to pytest discovery).